app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,31 @@
+class CategoriesController < ApplicationController
+  before_action :require_user
+
+  def new
+    @category = Category.new
+  end
+
+  def show
```

Comment:
Was able to access the category page of another user by using the category id. Might be worth mitigating similar to not being able to access another user's profile.

Reply:
Thanks for catching that. Should be fixed now

Reply:
Made a hotfix, should work now

Reply:
Thank you, Walid and Sunny.